### PR TITLE
changed the link of profile example to instructor path, imported dumm…

### PR DESCRIPTION
**Pull request by Jie-chelchel**
…yData into the InstructorHome index.js file

What have you done, 
changed the link of profile example to instructor path, imported dummyData into the InstructorHome index.js file

What files were changed
InstrutroHome/index.js
Home/RenderHomePage.js

Why was this change made?
we need dummydata to start with.
We can click on the link to go to the instructor page. 

[loom video] ![https://www.loom.com/share/5710865204eb4e6998218ccdd0fc0b05]

### CODE REVIEW

**Review by JoeyMBrown — APPROVED** [review]
Looks good lets get this merged! :)

**Review by fernando817mm — APPROVED** [review]

**Review by vasqueza91 — APPROVED** [review]
Everything looks good. Great job!

File: src/components/pages/InstructorHome/index.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import InstructorSidebar from './InstructorSidebar';
 import InstructorCalender from './InstructorCalender';
 import InstructorProfile from './InstructorProfile';
-
+import { dummyData } from '../../../dummyData';
```
Comment (bradlylewis) [inline src/components/pages/InstructorHome/index.js:5]: It's nice to go ahead and get this done. I wonder if, as we progress through the project, if we will convert to redux at some point?

File: src/components/pages/Home/RenderHomePage.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ function RenderHomePage(props) {
           approach components.
         </p>
         <p>
-          <Link to="/profile-list">Profiles Example</Link>
+          <Link to="/instructor">Instructor Path</Link>
```
Comment (MikeLikesCode) [inline src/components/pages/Home/RenderHomePage.js:16]: Let's get this merged as soon as possible!

**Review by chelseaceballos — APPROVED** [review]
Looks Good! Great Job!

File: src/components/pages/InstructorHome/index.js (hunk 1 above)
Comment (Jie-chelchel) [reply src/components/pages/InstructorHome/index.js:5]: Yes, once we get data from backend database, we will need to use redux to manage states.

**Pull request merged**